Caboodle/Email/Email.shared.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    /// <summary>
+    /// Email class for
+    /// </summary>
+    public static partial class Email
+    {
+        public static Func<string, string> BodyFormatter
+        {
+            get;
+            set;
+        }
```

Comment:
Put this on the same line.

Reply:
Wait, what is this for?

Reply:
I think we should omit this feature for now.